src/Apply/full_update.jl

```diff
@@ -0,0 +1,167 @@
+using KrylovKit: linsolve
+
+function full_update(
```

Comment:
Oh wow! Could you explain in the docstrings what the power and limitations are of these updates?

Reply:
Sure!